lib/board.rb

```diff
@@ -0,0 +1,69 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  #resets board
+  def reset!
+    @cells.clear
+    9.times do
+      @cells.push(" ")
+    end
+    @cells
+  end
+
+  #display board
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  #returns board position given user input
+  def position(position)
+    @cells[position.to_i-1]
+  end
+
+  #returns boolean if board is full or not
+  def full?
+    if @cells.include?(" ")
+      false
+    else
+      true
```

Comment:
Avoid returning boolean literals `true` and `false`. Instead, rely on the implicit return value of the boolean expression.
```
def full?
  @cells.include?(" ")
end
```